CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `Security`
 
+## Unreleased
+
```

Comment:
```suggestion
## 0.2.4
```
afther this pullrequest it done it shoud be released

Reply:
@pauljanns could you please fix the above thing? I cannot commit directly to develop.